src/main/java/bio/overture/songsearch/config/kafka/KafkaConfig.java

```diff
@@ -0,0 +1,53 @@
+package bio.overture.songsearch.config.kafka;
+
+
+import lombok.val;
+import org.apache.kafka.clients.producer.ProducerConfig;
+import org.apache.kafka.common.serialization.StringSerializer;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.kafka.core.DefaultKafkaProducerFactory;
+import org.springframework.kafka.core.KafkaTemplate;
+import org.springframework.kafka.core.ProducerFactory;
+
+import java.util.HashMap;
+import java.util.Map;
+
+
+@Configuration
+public class KafkaConfig {
+
+  @Value("${spring.kafka.bootstrap-servers}")
+  private String bootstrapServers;
+
+  @Value("${spring.kafka.template.default-topic}")
+  private String defaultTopic;
```

Comment:
should we give the topic a name descriptive of what we are using it for? like `workflow-automation-ingest-topic` or something?